File: src/main/java/com/tom/Hello.java
package com.tom;

public class Hello {
    public static void main(String[] args) {
//        System.out.println("Hello World !");
        Person p = new Person("Henry",66.5f,1.7f);
        p.hello();

        String s="abcde";
        System.out.println(s.charAt(3));
        System.out.println(s.length());
        System.out.println(s.substring(3));
        System.out.println(s.substring(1,4));
        p.weight=66.5f;
        p.height=1.7f;
        System.out.println(p.bmi());
        /*int age = 19;
        Integer age2 = 19;
        char c = '我';
        Character c2 = 'A';
        byte b =120;
        float weight =66.5f;//快捷鍵command+d 游標所在行複製到下一行
        float weight2 =66.5f;//快捷鍵command+delete 游標所在行刪除
        float height = 1.7f;
        boolean adult = true;
        boolean entrol=false;
        String name = "Tom";*/

        int score = 88;
        System.out.println(score<80 || score>90);
        char c = 'A';
        System.out.println(c>'a');

    }

}

File: src/main/kotlin/com/tom/Hello.kt
package com.tom

fun main() {
//    println("Hello Kotlin");
//    Human().hello()
    //版本控制視窗 command + 9
    //Git Commit PUSH 快選 control + v
    var s:String?="abcde" //null處理 ＋?
//    s=null
    println(s?.length)
    println(s?.get(3))
    println(s?.substring(3))
    val h = Human(weight = 66.5f,height = 1.7f)
      println(h.bmi())
    val score=88
    val c:Char ='A'
    println(c.toInt()>60)
    println(score>60)


/*    var age:Int = 25;
    age = age.plus(1)
    val weight =65.2
    val control:Boolean = true
    print("age = "+age)*/

}
class Human (var name:String="",var weight:Float,var height:Float){
    init {
        println("test $weight")//init在建構子完成的時候會自動執行
    }
//    constructor(name:String,weight: Float,height: Float):this(weight,height)//次要建構子 ，Kotlin建構子分主要與次要

    fun bmi() : Float{
        var bmi = weight/(height*height)
        return  bmi;
    }
    fun hello(){
        println("Hello Kotlin")
    }
}